src/energy_systems/heat_producers/gas_boiler.jl

```diff
@@ -6,26 +6,31 @@ buffer tank and en-/disabling the boiler when a threshold is reached, in additio
 overfill shutoff condition.
 """
 mutable struct GasBoiler <: ControlledComponent
-    uac::String
-    controller::Controller
-    sys_function::SystemFunction
+    uac::String # user address code -> e.g. location and affiliation of the units within the buildings
+    controller::Controller # control mechanism for the operating strategy
+    sys_function::SystemFunction # function of the component
 
-    input_interfaces::InterfaceMap
-    output_interfaces::InterfaceMap
+    input_interfaces::InterfaceMap # interfaces where gas boiler is reciever
+    output_interfaces::InterfaceMap # interfaces where gas boiler is source
 
     m_gas_in::Symbol
     m_heat_out::Symbol
 
     power::Float64
-    min_power_fraction::Float64
-    min_run_time::UInt
-    output_temperature::Temperature
+    load::String
+    #max_thermal_efficiency::Float64
+    max_consumable_gas::Float64
+    min_power_fraction::Float64 # Minimum amount of power needed for the component to run (?)
+    min_run_time::UInt # von state machine genutzt; min run time of component, e.g. several time steps
```

Comment:
In addition the comment on comments above, we prefer English as the language for documentation and comments.